README-it.md

```diff
@@ -0,0 +1,71 @@
+
+Per maggiori informazioni su come contribuire alla documentazione Kubernetes, vedi:
+
+* [Cominciare a contribuire](https://kubernetes.io/docs/contribute/start/)
+* [Vedere le modifiche localmente](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Utilizzare i template delle pagine](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Guida Stilistica della Documentazione](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Tradurre la Documentazione Kubernetes](https://kubernetes.io/docs/contribute/localization/)
+
+## Eseguire il sito Web localmente usando Docker
+
+Il modo consigliato per eseguire localmente il sito Web Kubernetes è eseguire un'immagine [Docker] (https://docker.com) specializzata che includa il generatore di siti Web statici [Hugo] (https://gohugo.io).
+
+> Se stai utilizzando Windows, avrai bisogno di alcuni strumenti aggiuntivi che puoi installare con [Chocolatey] (https://chocolatey.org). `choco install make`
+
+> Se preferisci eseguire il sito Web localmente senza Docker, vedi [Eseguire il sito Web localmente utilizzando Hugo](# running-the-site-local-using-hugo) di seguito.
+
+Se hai Docker [attivo e funzionante](https://www.docker.com/get-started), crea l'immagine Docker `kubernetes-hugo` localmente:
+
+```bash
+make docker-image
+```
+
+Dopo aver creato l'immagine, è possibile eseguire il sito Web localmente:
+
+```bash
+make docker-serve
+```
+
+Apri il tuo browser su http://localhost:1313 per visualizzare il sito Web. Mentre apporti modifiche ai file di origine, Hugo aggiorna il sito Web e forza un aggiornamento del browser.
+
+## Eseguire il sito Web localmente utilizzando Hugo
+
+Vedi la [documentazione ufficiale di Hugo] (https://gohugo.io/getting-started/installing/) per le istruzioni di installazione di Hugo. Assicurati di installare esattamente la versione di Hugo specificata dalla variabile d'ambiente `HUGO_VERSION` nel file [` netlify.toml`](netlify.toml#L9).
+
+Per eseguire il sito Web localmente dopo aver installato Hugo:
+
+```bash
+servire
+```
+
+Questo comando avvierà il server Hugo locale sulla porta 1313. Apri il tuo browser su http://localhost:1313 per visualizzare il sito Web. Mentre apporti modifiche ai file di origine, Hugo aggiorna il sito Web e forza un aggiornamento del browser.
```

Comment:
[1]

Reply:
this comment is not very clear to me. what did you meant?